auto_nag/scripts/severity_migration.py

```diff
@@ -0,0 +1,97 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from libmozdata.bugzilla import BugzillaProduct
+
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag.components import ComponentName
+
+
+class SeverityMigration(BzCleaner):
+    """Drop old severities to let teams retriage them.
+
+    TODO: This script is temporary; it should be removed once all old severities
+    are dropped.
+    """
+
+    no_bugmail = True
```

Comment:
Wdyt of sending the email in this case, so people will know which bugs have to be retriaged (otherwise they will have to go to the triage center)?

Reply:
It will be kind on noizy, the triage center could be enough. We could include these in the "to triage"/"no severity" emails instead. It will be one email per week instead of 10 +1.

Reply:
OK, sounds good.